### ISSUE TITLE
Neural Networks

### ISSUE THREAD

**Issue by cuppajoeman** [cuppajoeman/openmath#35]
# Prerequisite definitions/theorems we will need

Linear Algebra
* [ ] Any weighted summation is a dot product of two vectors

Matrix Calculus
* [ ] Gradients
* [ ] Jacobians
* [ ] Scalar Chain Rule
* [ ] Jacobian Chain Rule aka multivar chain rule

## Overall Structure of a neural network
	
* [ ] break down into each layer in the network
* [ ] break down how layers work
* [ ] connections between layers using neurons and activation functions
	* [ ] it's input is the weighted sum of the input and the weights + bias, RELU/sigmoid
* [ ] Cost Function - MSE (mean squared error) over our networks output and the label associated with the training sample over all samples

Backpropagation
* [ ] https://towardsdatascience.com/understanding-backpropagation-algorithm-7bb3aa2f95fd
